webpack/containers/Application/config.js

```diff
@@ -18,15 +18,18 @@ export const links = [
     component: WithOrganization(Subscriptions, '/subscriptions'),
   },
   {
+    text: 'Add Subscriptions',
     path: 'subscriptions/add',
     component: UpstreamSubscriptions,
   },
   {
+    text: 'Subscription Details',
     // eslint-disable-next-line no-useless-escape
     path: 'subscriptions/:id(\[0-9]*$\)',
     component: SubscriptionDetails,
   },
   {
+    text: 'Select Organization',
```

Comment:
The select organization page doesn't show the title `Select Organization`. It shows `RH Subscription` / `RH Repos` as the title. I believe that's not expected.
The other pages work great with the titles.. :+1: